crates/store/re_remote_store_types/proto/rerun/v0/common.proto

```diff
@@ -161,3 +161,15 @@ enum SparseFillStrategy {
     NONE = 0;
     LATEST_AT_GLOBAL = 1;
 }
+
+// Error codes for application level errors
+enum Code {
```

Comment:
Let's call this `ErrorCode` if it's only to be used for errors.

Otherwise `ReturnCode` is more clear, in which case I would just make `SUCCESS = 0`

Reply:
I was going back and forth on it as well, let's go with ``ErrorCode`` :)